src/pages/Cohort/Cohorts.tsx

```diff
@@ -103,8 +107,24 @@ export default function Cohorts() {
 
       renderCell: ({ row }) => {
         return (
-          <div className="flex items-center justify-center text-xs space-x-2 w-full h-full">
-            <Button size={ButtonSize.Small} outlined>
+          <div className="flex items-center justify-center w-full h-full space-x-2 text-xs">
+            <Button
+              size={ButtonSize.Small}
+              outlined
+              onClick={() =>
+                setDisplayCohortDetails({
+                  _id: row?.id,
+                  name: row?.name,
+                  description: row?.description,
+                  trainingStartDate: row?.trainingStartDate,
+                  stages: row?.stages,
+                  applicants: row?.applicants,
+                  coaches: row?.coaches,
+                  forms: row?.forms,
+                  trainees: row?.trainees,
+                })
```

Comment:
I'd suggest something similar here. If the use gets to click the view button, it means that the cohort details were available in the first place to be able to see that button. That's why I think these optional chaining operators are redudant.
```suggestion
 setDisplayCohortDetails({
                  _id: row._id,
                  name: row.name,
                  description: row.description,
                  trainingStartDate: row.trainingStartDate,
                  stages: row.stages,
                  applicants: row.applicants,
                  coaches: row.coaches,
                  forms: row.forms,
                  trainees: row.trainees,
                })
```